contracts/vault-managers/VaultManagerChainlinkMainAsset.sol

```diff
@@ -24,6 +24,8 @@ contract VaultManagerChainlinkMainAsset is ReentrancyGuard {
     ChainlinkedOracleSimple public immutable oracle;
     uint public constant ORACLE_TYPE = 5;
 
+    uint public constant Q112 = 2 ** 112;
```

Comment:
Why are the changes to `LiquidationTriggerChainlinkMainAsset.sol` and `VaultManagerChainlinkMainAsset.sol` included in this PR?

Reply:
Since these contracts have been updated in this PR

Reply:
And why are they updated in this PR?

Reply:
Since the implementation of Curve LP oracle depends on these contracts. And as you mentioned before it's really nice to follow single standard for oracle wrappers.

Reply:
Dependent changes should ideally go into separate PRs, so that the structure and order of changes are transparent. Thats what I wanted to say.